xml/issue3753.xml

```diff
@@ -239,6 +238,108 @@ entities.</del>
 </li>
 </ol>
 
+</blockquote>
+
+<note>2022-11-09; Ben Craig and Tomasz provides improved wording</note>
+<p>
+This new resolution merges definition of freestanding item for entity in macro int bullet lists.
+It still n addresses both 3754 and LWG <iref ref="3815"/>.
+</p>
+
+<p>
+This resolves ballot comment GB-75.
+It also partially addresses GB-130 (along with LWG <iref ref="3814"/>).
+</p>
+
+</discussion>
+
+<resolution>
+
+<p>
+This wording is relative to <paper num="N4917"/>.
+</p>
+
+<ol>
+<li><p>Modify 16.3.3.6 <a href="https://wg21.link/freestanding.entity">[freestanding.entity]</a> as indicated:</p>
+
+<blockquote class="note">
+<p>
+[<i>Drafting note:</i> Replace the section name [freestanding.entity] by [freestanding.item] throughout the working draft]
+</p>
+</blockquote>
+
+<blockquote>
+<p>
+<b>16.3.3.6 Freestanding <del>entities</del><ins>items</ins> [freestanding.<ins>item</ins><del>entity</del>]</b>
+<p/>
+-1- A <i>freestanding <del>entity</del><ins>item</ins></i> is <del>a declaration</del><ins>an entity</ins> or
+macro definition that is present in a freestanding implementation and a hosted implementation.
+<p/>
+-2- Unless otherwise specified, the requirements on freestanding <del>entities</del><ins>items, except namespaces,</ins> on a freestanding implementation are the same as the corresponding requirements in a hosted implementation.
+<p/>
+<ins>[<i>Note</i>: This implies that freestanding item enumerations have the same enumerators on freestanding implementations and hosted implementations.  
+	 Futhermore, class types have the same deduction guides and members on freestanding implementations and hosted implementations. — <i>end note</i>]</ins>
+<p/>
+<ins>-3- An entity is a freestanding item, if it is:</ins> 
+<ol style="list-style-type: none">
+<li><p><ins>(3.1) &mdash; introduced by a declaration in the header synopsis, and</ins></p>
+<ol style="list-style-type: none">
+<li><p><ins>(3.1.1) &mdash; the declaration is followed by a comment that includes <i>freestanding</i>, or</ins></p></li>
+<li><p><ins>(3.1.2) &mdash; the header synopsis begins with a comment that includes <i>all freestanding;</i></ins></p></li>
```

Comment:
This semi-colon should not be in the italics. I'm not sure it should be there at all.

Reply:
I used it because it is a nested sentence, so if bullets would be removed it would be unclear if 3.2 applies to 3.1.2 or 3.1